From f1921384d48a40684bfc06b9813073797affa10a Mon Sep 17 00:00:00 2001
From: Gergely Nagy <algernon@madhouse-project.org>
Date: Mon, 12 Aug 2013 09:41:41 +0200
Subject: [PATCH] Fix TexInfo error

This fixes buiilding the docs with recent texinfo, where some warnings
became errors. (Debian bug #712332).

Patch from Daniel Schepler <dschepler@gmail.com>.
---
 doc/aalib.texinfo | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/doc/aalib.texinfo b/doc/aalib.texinfo
index 42e1612..2a4b514 100644
--- a/doc/aalib.texinfo
+++ b/doc/aalib.texinfo
@@ -29,11 +29,11 @@ this permission notice are preserved on all copies.
 
 @titlepage
 
-@title{AA-lib @value{VERSION}}
-@subtitle{An ascii-art library}
-@subtitle{API-DESCRIPTION}
+@title AA-lib @value{VERSION}
+@subtitle An ascii-art library
+@subtitle API-DESCRIPTION
 
-@author{Jan Hubi@v cka & Kamil Toman}
+@author Jan Hubi@v cka & Kamil Toman
 @tex
 Dukelsk\'ych bojovn\'\i ku 1944 
 @end tex
@@ -1318,7 +1318,7 @@ To write new driver is quite easy. You need to implement just few very basic
 functions (like initialization, drawing to screen etc...) and register it in
 the driver registry. There is separate drivers for screen, mouse and keyboard.
 
-@subsection Display driver
+@section Display driver
 Display driver is the most significant driver (of course) so it is recomended
 to implement it first. Its structure is as follows:
 
-- 
2.0.1

